specification/search/data-plane/Microsoft.Azure.Search.Service/preview/2017-11-11-preview/searchservice.json

```diff
@@ -4197,21 +4201,21 @@
 			},
 			"context": {
 				"type": "string",
-				"description": "The context of the skill."
+				"description": "Represents the level at which operations take place, such as the document root or document path (for example, document/sentiment or document/content)."
```

Comment:
document root or document content (for example, /document or /document/content)

Reply:
Fixed